renku/__init__.py

```diff
@@ -19,10 +19,23 @@
 
 from __future__ import absolute_import, print_function
 
+import os
+
 from .api import LocalClient
 from .version import __version__
 
 __all__ = (
     'LocalClient',
     '__version__',
 )
+
+# Wait for the VS Code debugger to attach if requested
+VSCODE_DEBUG = os.environ.get('VSCODE_DEBUG') == '1'
```

Comment:
Editor specific stuff should be kept in the editor. You can easily configure your editor to do this automatically, without putting it in the codebase.